app/models/manageiq/providers/nsxt/manager_mixin.rb

```diff
@@ -1,5 +1,3 @@
-require 'rest-client'
```

Comment:
RestClient wasn't referenced here at all, might have been a holdover from before the NsxtClient was extracted out